src/node_crypto.cc

```diff
@@ -3256,14 +3256,11 @@ bool CipherBase::SetAAD(const char* data, unsigned int len) {
   if (ctx_ == nullptr || !IsAuthenticatedMode())
     return false;
   int outlen;
-  if (!EVP_CipherUpdate(ctx_,
-                        nullptr,
-                        &outlen,
-                        reinterpret_cast<const unsigned char*>(data),
-                        len)) {
-    return false;
-  }
-  return true;
+  return !!EVP_CipherUpdate(ctx_,
```

Comment:
Nit: I'd prefer an explicit cast or tertiary operator. Not a fan of implicit casts: 

`return EVP_CipherUpdate(...) ? true : false;`

Reply:
Can't it just return the result of `EVP_CipherUpdate`? That's what `CipherBase::Update` does.

Reply:
I changed it to `== 1` in #18138 to avoid the implicit cast.

Reply:
@fhinkel I got rid of the `!!`, PTAL